docs/api/index.md

```diff
@@ -243,6 +243,23 @@ You cannot use this syntax, when using Vitest as [type checker](/guide/testing-t
   // ✓ add(2, 1) -> 3
   ```
 
+  You can also access template string table.
+    * First row of variable name column headings separated with `|`
+    * One or more subsequent rows of data supplied as template literal expressions using ${value} syntax.
+
+    ```ts
+    test.each`
+    a    | b    | expected
```

Comment:
Let's make it more readable, can you add more spaces, so `|` character visually forms a table?